tests/auto/util/tst_util.cpp

```
@@ -154,6 +154,17 @@ private Q_SLOTS:
   void updateEnvSetsExpectedVars();
   void updateEnvEmptyCustomCharsetFallsBackToAllChars();
   void updateEnvWslenvContainsRequiredVars();
+  void gpgErrorMessageKeyExpiredStatusToken();
+  void gpgErrorMessageKeyRevokedStatusToken();
+  void gpgErrorMessageNoPubkeyStatusToken();
+  void gpgErrorMessageInvRecpStatusToken();
+  void gpgErrorMessageFailureStatusToken();
+  void gpgErrorMessageKeyExpiredFallback();
+  void gpgErrorMessageRevokedFallback();
+  void gpgErrorMessageNoPubkeyFallback();
+  void gpgErrorMessageEncryptionFailedFallback();
```

Cppcheck (2.20.0)

[error] There is an unknown macro here somewhere. Configuration is required. If slots is a macro then please configure it.

(unknownMacro)